content/doc/book/managing/nodes.adoc

```diff
@@ -14,23 +14,23 @@ endif::[]
 
 = Managing Nodes
 
-== Launch Java Web Start slave agent via Windows Scheduler
+== Launch Java Web Start agent via Windows Scheduler
 
-If you are having trouble getting the Java Web Start agent installed as a Windows service (i.e., you followed https://wiki.jenkins.io/display/JENKINS/Installing+Jenkins+as+a+Windows+service[the instructions on installing the slave as a service here] but it didn't work), an alternative method of starting the service automatically when Windows starts is to use the Windows Scheduler. 
+If you are having trouble getting the Java Web Start agent installed as a Windows service (i.e., you followed https://wiki.jenkins.io/display/JENKINS/Installing+Jenkins+as+a+Windows+service[the instructions on installing the agent as a service here] but it didn't work), an alternative method of starting the service automatically when Windows starts is to use the Windows Scheduler. 
 
 We take advantage of the Windows Scheduler's ability to run command at system startup
 
-. Configure your node to use the "Launch slave agents via Java Web Start" launch method
+. Configure your node to use the "Launch agents via Java Web Start" launch method
```

Comment:
The term "Java Web Start" is a source of confusion for Java 11 users because it is not a standard part of Java 11.  We've switched the terminology to refer to "inbound agents" rather than "Java Web Start" or "JNLP" agents.  That terminology is more precise and will reduce confusion long-term.
```suggestion
. Configure your node to use the "Launch agents by connecting it to the master" launch method
```